data/lib/src/network/linshare_http_client.dart

```diff
@@ -233,10 +233,13 @@ class LinShareHttpClient {
     }
   }
 
-  WorkGroupNodeDto _convertToWorkGroupNodeChild(Map<String, dynamic> nodeChildJson) =>
-      nodeChildJson['type'] == WorkGroupNodeType.DOCUMENT.value
-          ? WorkGroupDocumentDto.fromJson(nodeChildJson)
-          : WorkGroupNodeFolderDto.fromJson(nodeChildJson);
+  WorkGroupNodeDto _convertToWorkGroupNodeChild(Map<String, dynamic> nodeChildJson) {
+    if (nodeChildJson['type'] == WorkGroupNodeType.FOLDER.value) {
+      return WorkGroupNodeFolderDto.fromJson(nodeChildJson);
+    }
+
+    return WorkGroupDocumentDto.fromJson(nodeChildJson);
```

Comment:
It is the same for `DOCUMENT` and `DOCUMENT_REVISION`?

Reply:
yes same data